masterlist.yaml

```
@@ -3754,6 +3754,29 @@ plugins:
         crc: 0x74BE02DE
         util: '[FO4Edit v4.0.2f](https://www.nexusmods.com/fallout4/mods/2737)'
         nav: 1
+
+# Subway Runner
+  - name: 'subwayrunnner(nodynamic|dynamiclighting)\.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/18639' ]
+  - name: 'subwayrunnnerdynamiclighting.esp'
+    dirty:
+      - <<: *quickClean
+        crc: 0xEB0209A0
+        util: '[FO4Edit v4.0.3](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 12
+    clean:
+      - crc: 0x99CFDDBD
+        util: 'FO4Edit v4.0.3'
+  - name: 'subwayrunnnernodynamic.esp'
```

It doesn't really make much difference as you're still using the same number of line.
But it's up to you as either way will work.

```suggestion
  - name: 'subwayrunnnerdynamiclighting.esp'
    url: [ 'https://www.nexusmods.com/fallout4/mods/18639' ]
    dirty:
      - <<: *quickClean
        crc: 0xEB0209A0
        util: '[FO4Edit v4.0.3](https://www.nexusmods.com/fallout4/mods/2737)'
        itm: 12
    clean:
      - crc: 0x99CFDDBD
        util: 'FO4Edit v4.0.3'
  - name: 'subwayrunnnernodynamic.esp'
    url: [ 'https://www.nexusmods.com/fallout4/mods/18639' ]
```

---

Should be resolved, gave them both their own individual line.